.circleci/config.yml

```diff
@@ -55,10 +54,10 @@ jobs:
             echo 'export NVM_DIR="/opt/circleci/.nvm"' >> $BASH_ENV
             echo "[ -s \"$NVM_DIR/nvm.sh\" ] && . \"$NVM_DIR/nvm.sh\"" >> $BASH_ENV
       - node/install-yarn
-      - cypress/install:
-          cache-key: cache-{{ arch }}-{{ .Branch }}-{{ checksum "tdrs-frontend/package.json" }}
-          working_directory: tdrs-frontend
-          yarn: true
+      - node/install-packages:
```

Comment:
This command does the same thing, but doesn't cache the Cypress binary by default - which was causing an issue before.